src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -1514,6 +1517,108 @@ def handle_assign_role_submission_event(
     )
 
 
+def handle_create_task_command(
+    ack: Ack,
+    body,
+    dict,
+    client: WebClient,
+    context: BoltContext,
+    db_session: Session,
+) -> None:
+    """Displays a modal for task creation."""
+    ack()
+    if context["subject"].type == CaseSubjects.case:
+        raise CommandError("Command is not currently available for cases.")
+
+    participants = participant_service.get_all_by_incident_id(
+        db_session=db_session, incident_id=context["subject"].id
+    )
+
+    incident = incident_service.get(db_session=db_session, incident_id=context["subject"].id)
+
+    contact_plugin = plugin_service.get_active_instance(
+        db_session=db_session, project_id=incident.project.id, plugin_type="contact"
+    )
+    if not contact_plugin:
+        raise CommandError(
+            "Contact plugin is not enabled. Unable to list participants.",
+        )
+
+    active_participants = [p for p in participants if p.active_roles]
+    participant_list = []
+
+    for participant in active_participants:
+        participant_email = participant.individual.email
+        participant_info = contact_plugin.instance.get(participant_email, db_session=db_session)
+        participant_name = participant_info.get("fullname", participant.individual.email)
+        participant_list.append({"text": participant_name, "value": participant_email})
+
+    blocks = [
+        static_select_block(
+            label="Participant",
```

Comment:
suggestion: maybe `Assignee` is more descriptive?

Reply:
Done.